plugins/prowl/__init__.py

```diff
@@ -55,10 +52,15 @@ def __call__(self, event='', description='', priority=None, url=None, apikey=Non
         if url:
             data['url'] = url[:512]
         try:
-            conn = http.client.HTTPSConnection(self._host, timeout=4)
-            conn.request("POST", self._api, urllib.parse.urlencode(data), headers)
-            resp = conn.getresponse()
-            conn.close()
+            apiurl='https://api.prowlapp.com/publicapi/add'
+            data = urllib.parse.urlencode(data)
+            data = data.encode('utf-8')
+            req = urllib.request.Request(apiurl, data)
+            req.add_header = ('User-Agent','SmartHome.py')
+            req.add_header = ('Content-Type','application/x-www-form-urlencoded;charset=utf-8')
+
+            logger.debug("lala");
```

Comment:
This debugging stuff could be changed to something more interesting or could be removed completely ;-)